homeassistant/components/simplefin/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""The SimpleFIN integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from simplefin4py import SimpleFin
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_ACCESS_URL
+from .coordinator import SimpleFinDataUpdateCoordinator
+
+PLATFORMS: list[str] = [Platform.SENSOR]
+
+
+type SimpleFinConfigEntry = ConfigEntry[SimpleFinData]
+
+
+@dataclass
+class SimpleFinData:
+    """Class to hold SimpleFIN data."""
+
+    sf_client: SimpleFin
+    sf_coordinator: SimpleFinDataUpdateCoordinator
```

Comment:
You're not really using the client at the moment, might as well just only keep the coordinator in there